api.go

```diff
@@ -61,3 +62,18 @@ func (es *ExploreServer) apiGetBlockData(start types.BlockHeight, end types.Bloc
 
 	return blockSummaries, err
 }
+
+// apiGetHash queries the running instance of siad and returns the raw
+// data to the calling function. The json data could decode into
+// multiple types of structures, all of which *should* have the
+// ResponseType field as a string for identifying the type of data
```

Comment:
The docstrings could do with some shortening. You're expressing a lot of implicit/repetitive things like "returns ... to the calling function" and "decode into multiple types ... for identifying the type of data"

@DavidVorick might disagree (with his gi-normous docstrings), but I think this could easily convey the same message phrase as:
"apiGetHash queries siad and returns the raw data. The json data can be decoded based on the ResponseType field"